OpenQA/Isotovideo/CommandHandler.pm

```diff
@@ -17,7 +19,10 @@ use constant AUTOINST_STATUSFILE => 'autoinst-status.json';
 
 
 # io handles for sending data to command server and backend
-has [qw(test_fd cmd_srv_fd backend_fd backend_out_fd answer_fd)] => undef;
+has [qw(test_fd cmd_srv_fd backend backend_fd backend_out_fd answer_fd)] => undef;
```

Comment:
`undef` is the default value anyway, even without assigning it.

Reply:
ok, but this was there before